examples/visualization/plot_study.ipynb

```diff
@@ -0,0 +1,272 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Visualizing High-dimensional Parameter Relationships\n",
+    "\n",
+    "This notebook demonstrates various visualizations of studies in Optuna.\n",
+    "The hyperparameters of a neural network trained to classify images are optimized and the resulting study is then visualized using these features.\n",
```

Comment:
This document contains both `hyperparameter` and `hyper-parameter`. If you don't have a specific reason, we can use only `hyperparameter` like other documents.

Reply:
Thank you for the catch. Just fixed it accordingly.